libs/core/include/core/commandline/vt100.hpp

```diff
@@ -25,56 +25,17 @@ namespace fetch {
 namespace commandline {
 namespace VT100 {
 
-uint16_t    ColorFromString(std::string name);
 std::string GetColor(int const &fg, int const &bg);
-std::string GetColor(std::string const &f, std::string const &b);
 
-inline static constexpr char const *DefaultAttributes()
+inline constexpr char const *DefaultAttributes()
```

Comment:
Double check but I think that we can remove almost all of these VT100 files

Reply:
I had checked at the time. The fragments I left are used elsewhere, I think.